controllers/managedmcg_controller_test.go

```diff
@@ -138,10 +138,35 @@ func newOcsCsvFake() opv1a1.ClusterServiceVersion {
 	return ocscsv
 }
 
+// Remove temporory files/folder or other data after finishing test.
 func cleanup() {
 	os.Remove(CustomerNotificationHTMLPath)
 }
 
+func miscTest(t *testing.T, rec *ManagedMCGReconciler) {
+	t.Helper()
```

Comment:
I created this function to avoid having one bif function doing everything and becoming difficult to read/understand.
This t.Helper was needed only to satisfy lint error.
Anyway I am removing this function and putting these test in main test function only.